src/views/components/NewsletterList.js

```
@@ -0,0 +1,117 @@
+                let response = await window.http.get(`/user/my/newsletters`)
+                this.newsletters = response.data.results.data;
+            } catch (error) {
+                if (error.response) {
+                    throw new Error(error.response.data.message);
+                }
+                throw new Error(error.message);
+            }
+        },
+        formatDate: function (value) {
+            if (!value) return ''
+            if (isNaN(value)) return 'Invalid date';
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)